README.md

```diff
@@ -1,3 +1,25 @@
 # TypeDoc homepage
-
 Holds the contents of http://typedoc.org.
+GitHub pages renders the website from the `gh-pages` branch using [Jekyll](https://jekyllrb.com/).
+
+## Writing guides
+Guides are stored in the `_guides/` directory and written in Markdown. Each file should contain the following header section for Jekyl with an appropriate title and menu order.
+
+```
+---
+layout: 'guide'
+title: 'Installation'
+menuOrder: 1
+---
+```
+
+## Running Jekyll locally
+You can run Jekyll on a computer with Ruby 2.1 or higher installed and the `bundle` gem.
```

Comment:
`bundle` will complain if your Ruby version doesn't match the version specified in `Gemfile.lock`, so this should probably be Ruby 2.5.3